symbols/arm9.yml

```diff
@@ -3153,6 +3184,20 @@ arm9:
         
         r0: animation_control
         return: wan_header
+    - name: SetAndPlayAnimationForAnimationControl
+      address:
+        EU: 0x201C4B4
+      description: |-
+        Set the animation to play with the animation control, and start it.
+        
+        r0: animation_control
+        r1: animation key (either an animation or animation group depending on the type of sprite and if it does have animation group with this animation key as index)
+        r2: direction_id (unsure) (the key to the wan_animation in itself, only used when animation key represent a wan_animation_group)
+        r3: ?
+        stack[0]: low_palette_pos
+        stack[1] (0x4): ?
+        stack[2] (0x8): ?
+        stack[3] (0xC): ?
```

Comment:
This last parameter isn't given in the function header you defined. It should either be removed here, or added to the function header.